packages/devtools/src/devtoolsdriver.ts

```diff
@@ -30,11 +31,9 @@ export default class DevToolsDriver {
     currentFrame?: Page
     currentWindowHandle?: string
     currentFrameUrl?: string
-
+    activeListeners: {emitter: EventEmitter, eventName: string, boundHandler: any}[] = []
```

Comment:
Can we define this as `ActiveListener` type in `types.ts` for better readability?

Reply:
done